src/tasks/webpack.viewSandbox.config.js

```diff
@@ -7,19 +7,16 @@ module.exports = {
   entry: {
     viewSandbox: files.VIEW_SANDBOX_JS_PATH
   },
+  mode: 'production',
   // AJV needs json-loader.
   module: {
     // Without this webpack will log a warning.
     // https://github.com/epoberezkin/ajv/pull/288#issuecomment-253286408
     noParse: /node_modules\/ajv\/dist\/ajv.bundle.js/,
-    loaders: [
```

Comment:
`json-loader` is no longer needed. Webpack loads JSON files natively now. `loaders` was replaced by `rules`.